src/sentry/static/sentry/app/views/organizationGroupDetails/groupTagValues.tsx

```diff
@@ -40,11 +42,16 @@ class GroupTagValues extends AsyncComponent<
   Props & AsyncComponent['props'],
   State & AsyncComponent['state']
 > {
-  getEndpoints(): [string, string][] {
+  getEndpoints(): [string, string, any?][] {
+    const {environments: environment} = this.props;
     const {groupId, tagKey} = this.props.params;
     return [
       ['tag', `/issues/${groupId}/tags/${tagKey}/`],
-      ['tagValueList', `/issues/${groupId}/tags/${tagKey}/values/`],
+      [
+        'tagValueList',
+        `/issues/${groupId}/tags/${tagKey}/values/`,
+        {query: {environment}},
```

Comment:
Will this work with pagination?

Reply:
@billyvg It does seem to work with both environment and "cursor" parameters.